test/tic_tac_toe/unit/game_presenter_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GamePresenterTest do
+  use ExUnit.Case
+
+  test "renders a human-v-human new game" do
+    #wondering if I should have unit tests for this as the UI is partially tested
```

Comment:
I think my interpretation of a Presenter is different to yours. When we've used it on projects, a presenter is something that takes an object and wraps the adds presentation logic to them.

For example, a `User` object may have methods like first_name, last_name, etc. but a UserPresenter may take that object and provide methods like `full_name`, or `date_or_birth`  which would format the date to 'DD/MM/YYYY' for example.